mcrcon.c

```diff
@@ -304,7 +304,7 @@ int net_connect(const char *host, const char *port)
 {
 	int sd;
 
-	struct addrinfo hints = {0};
+	struct addrinfo hints = {.ai_family = 0};
```

Comment:
Yep, you guessed it!

```
gcc -std=gnu99 -Wall -Wextra -Wpedantic -Os -s -fstack-protector-strong -o mcrcon mcrcon.c
mcrcon.c:307:28: warning: missing field 'ai_family' initializer [-Wmissing-field-initializers]
        struct addrinfo hints = {0};
                                  ^
1 warning generated.
ld: warning: option -s is obsolete and being ignored
```

The above is with a system "gcc" via Xcode:

``` bash
$ gcc --version
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Apple LLVM version 8.0.0 (clang-800.0.42.1)
Target: x86_64-apple-darwin15.6.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

I'll get a revision with `memset` pushed shortly.